datastore/sync_entity.go

```diff
@@ -745,16 +779,27 @@ func CreateDBSyncEntity(entity *sync_pb.SyncEntity, cacheGUID *string, clientID
 		originatorClientItemID = entity.IdString
 	}
 
-	now := aws.Int64(utils.UnixMilli(time.Now()))
+	if dataType == historyTypeID {
```

Comment:
Please add a comment above this line for what we're doing for history and delete directive here.